scripts/publish_layers.sh

```diff
@@ -76,12 +76,12 @@ elif [ "$ARCHITECTURE" == "arm64" ]; then
 else
     ARCHITECTURE_MESSAGE="both architectures"
 fi
-read -p "Ready to publish Lambda Extension version $VERSION (for $ARCHITECTURE_MESSAGE) to regions ${REGIONS[*]} (y/n)?" CONT
+# read -p "Ready to publish Lambda Extension version $VERSION (for $ARCHITECTURE_MESSAGE) to regions ${REGIONS[*]} (y/n)?" CONT
```

Comment:
Seems like changes to this file weren't meant to be included in the commit?

Reply:
Yes this is correct, I needed to remove this to move faster 🙃